patches/chrome-android-java-src-org-chromium-chrome-browser-page_info-ConnectionInfoPopup.java.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/android/java/src/org/chromium/chrome/browser/page_info/ConnectionInfoPopup.java b/chrome/android/java/src/org/chromium/chrome/browser/page_info/ConnectionInfoPopup.java
+index 652b6a4bf72cee2aed636d5392eac57bfb1f61e8..3e8a6e07dd049f5fa2531b352e7299559a0c1ee3 100644
+--- a/chrome/android/java/src/org/chromium/chrome/browser/page_info/ConnectionInfoPopup.java
++++ b/chrome/android/java/src/org/chromium/chrome/browser/page_info/ConnectionInfoPopup.java
+@@ -174,7 +174,7 @@ public class ConnectionInfoPopup implements OnClickListener, ModalDialogProperti
+     @CalledByNative
+     private void addMoreInfoLink(String linkText) {
+         mMoreInfoLink = new TextView(mContext);
+-        mLinkUrl = HELP_URL;
```

Comment:
we should be able to handle this with `brave_static_redirect_network_delegate_helper.cc`